allennlp/nn/util.py

```diff
@@ -113,6 +114,10 @@ def arrays_to_variables(data_structure: Dict[str, Union[dict, numpy.ndarray]],
         using this function. This is useful during inference for passing
         tensors representing a single example to a Pytorch model which
         would otherwise not have a batch dimension.
+    for_training : ``bool``, optional (default = ``True``)
```

Comment:
The trouble with using `volatile` as the name here is that it has no relation to anything.  `volatile` to me is about thread safety, not about whether the tensor will allow gradient computation.  So `volatile` is an incredibly unhelpful name for this parameter.  On the other hand, the trouble with changing it is that if you're looking for how to pass the `volatile=True` flag in pytorch to the variable creation code, it's not as easy to find what you're looking for.  That's why I added in the comment.